src/actions/start/start.ts

```diff
@@ -40,16 +45,18 @@ export async function start(args: ReadonlyArray<string>): Promise<void> {
   if (!constants.mnemonic) {
     throw new Error("The FAUCET_MNEMONIC environment variable is not set");
   }
-  await setSecretAndCreateIdentities(profile, constants.mnemonic);
   const signer = new MultiChainSigner(profile);
-
   console.log("Connecting to blockchain ...");
   const connection = (await signer.addChain(chainConnector(codec, blockchainBaseUrl))).connection;
 
   const connectedChainId = connection.chainId();
   console.log(`Connected to network: ${connectedChainId}`);
 
-  const accounts = await accountsOfFirstChain(signer);
+  setFractionalDigits(codecDefaultFractionalDigits(codec));
```

Comment:
Why are you storing this value somewhere? Can't you just use `codecDefaultFractionalDigits(codec)` in every place you need it?

Reply:
I would have to pass it to `creditAmount` each time, `fractionalDigits` is more a global variable that something that should be calculated each time